datasets/GPM_3IMERGDF.data.mdx

```diff
@@ -0,0 +1,74 @@
+---
+id: GPM_3IMERGDF.v07
+name: "GPM IMERG Daily Precipitation"
+description: "GPM IMERG Final Precipitation L3 1 day 0.1 degree x 0.1 degree"
+media:
+  src: ::file ./gpmimergdaily.png
+  alt: CMIP6 Near-Surface Air Temperature Screenshot
+  author:
+    name: NASA
+    url:
+taxonomy:
+  - name: Topics
+    values:
+      - Climate
+layers:
+  - id: GPM_3IMERGDF.v07
+    type: cmr
+    stacCol: GPM_3IMERGDF
+    tileApiEndpoint: "https://dev-titiler-cmr.delta-backend.com/WebMercatorQuad/tilejson.json"
+    name: GPM IMERG Final Precipitation L3 1 day 0.1 degree x 0.1 degree
+    description: "GPM Level 3 IMERG Final Daily 10 x 10 km (GPM_3IMERGDF) accumulated precipitation"
+    time_density: day
+    zoomExtent:
+      - 0
+      - 20
+    sourceParams:
+      resampling: bilinear
+      variable: precipitation
+      colormap_name: gnbu
+      rescale:
+        - 0
+        - 46
+      maxzoom: 12
+      concept_id: C2723754864-GES_DISC
+      backend: xarray
```

Comment:
```suggestion
      backend: xarray
      tile_scale: 3
```

I think we should do the same thing as in https://github.com/US-GHG-Center/veda-config-ghg/pull/630 to make the data look its best (crispiest) in demos, even though the user / information value is perhaps limited.